### PR TITLE
Use Tanstack data to display assets badge lists

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: apps/desktop-wallet/src/features/assetsLists/AssetsBadgesList.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  ...badgeProps
+}: AssetsLogosListProps) => {
+  const { t } = useTranslation()
+  const { data: tokensBalances } = useAddressesTokensBalances(addressHash)
+  const { data: sortedFTs, isLoading: isLoadingSortedFTs } = useAddressesSortedFungibleTokens(addressHash)
+  const { data: NFTs, isLoading: isLoadingNFTs } = useAddressesNfts(addressHash)
+  const { data: nonStandardTokens, isLoading: isLoadingNonStandardTokens } =
+    useAddressesUnlistedNonStandardTokenIds(addressHash)
+
+  const displayedFTs = sortedFTs.slice(0, maxDisplayedAssets)
```
Comment (mvaivre) [inline apps/desktop-wallet/src/features/assetsLists/AssetsBadgesList.tsx:50]: After using it, what do you think about having `FTs` and `NFTs`?
I'm not yet convinced, I find it easy to confuse variables. Maybe I just need to get used to it.

Comment (nop33) [reply apps/desktop-wallet/src/features/assetsLists/AssetsBadgesList.tsx:50]: Funny, I just sent a msg on slack to discuss this, without seeing this message 😆  🧠 ⚡ 🧠  Let's continue the discussion there!

Comment (nop33) [reply apps/desktop-wallet/src/features/assetsLists/AssetsBadgesList.tsx:50]: So, I am in favor of the list
- `listedFT`
- `unlistedFT`
- `NFT`
- `NST`

I wouldn't go as far as to name the first 2 `LFT` and `UFT`. WDYT?

Comment (nop33) [reply apps/desktop-wallet/src/features/assetsLists/AssetsBadgesList.tsx:50]: btw, the following open PR implements the above proposal, you can check it out there to get a feeling for it: https://github.com/alephium/alephium-frontend/pull/791

**Review by mvaivre — APPROVED** [review]
LGTM!

**Pull request merged**